crates/rolldown_tracing/Cargo.toml

```diff
@@ -21,4 +21,4 @@ test = false
 [dependencies]
 tracing = { workspace = true, features = ["valuable"] }
 tracing-chrome = { workspace = true }
-tracing-subscriber = { workspace = true, features = ["env-filter", "json", "valuable"] }
```

Comment:
I'm not seeing `json` and `valuable` used any where, disabling them is not breaking anything?